ski_conditions/apps/app_scraping/views.py

```diff
@@ -9,5 +9,22 @@ def index(request):
     template = loader.get_template('app_scraping/index.html')
     context = {
         'resort_list': output,
-        }
+    }
     return HttpResponse(template.render(context, request))
+
+
+def twitter(request):
```

Comment:
What’s the purpose of this view? Why not use a template?

Reply:
Moved twitter feed embedding html into a template.